### DISCUSSION TITLE
Kube-ovn 1.10.X upgrade

### DISCUSSION THREAD

**Discussion by Kovendan03** [kubeovn/kube-ovn#2059] [General]
Hi Team,

I am planning to upgrade kube-ovn from 1.9.2 to 1.10.x (latest version). Currently, we have the options to run the cleanup.sh shell script, which removes all the items created. 

Is there a way we can upgrade from old to latest version with existing network data retained? If so, can you please share the upgrade steps.

**Comment by oilbeater** [comment]
The `cleanup.sh` will clean all annotations,crds and db files, so it's not suitable for upgrade. We usually compare the yaml diff between two `install.sh` and then do a patch update to each component.
